README.md

```diff
@@ -354,6 +354,25 @@ Finally, edit the workfolw file `.github/workflows/action.yml` in your forked re
 After you commited this change, you will see GitHub Actions triggered to build and deploy all the apps in the repo to your Azure Spring Cloud instance.
 ![](./media/automate-deployments-using-github-actions.jpg)
 
+## Reconfigure the microservices to use Azure KeyVault to get the database configuration
+
+If you want to use Azure KeyVault to get the database configuration you need
+to take the following steps:
+
+1. Create an Azure KeyVault
```

Comment:
@mnriem - shall we include a reference to the previous section where we are explaining how to do this?